### DISCUSSION TITLE
Electricity Maps Data Portal: Get access to granular carbon emissions data

### DISCUSSION THREAD

**Discussion by Celineleopoul** [Green-Software-Foundation/discussions#48] [News]
Electricity Maps has developed a [data portal](https://www.electricitymaps.com/data-portal) that empowers companies to do more granular carbon accounting.

As part of our ongoing commitment to decarbonizing the electricity grid, Electricity Maps empowers businesses to make informed decisions and drive meaningful change in the fight against climate change. Decarbonizing the grid is extremely complex and a lot of factors come into play. Companies around the globe are integral to this transformation. We want to support this transition by enabling companies to do more granular carbon accounting. Electricity Maps is now offering access to open data on carbon emissions from electricity generation for more than 50+ countries all over the world. This data will enable businesses to accurately measure and monitor their scope 2 carbon emissions and drive sustainable practices.

[Access the data portal here.](https://www.electricitymaps.com/data-portal)

**About the portal**

The data portal gives companies and researchers free access to historical electricity data. The data includes location-based carbon intensity based on both direct emissions and life cycle analysis (LCA) of electricity generation - and at different granular levels such as hourly, daily, and monthly.

It is currently difficult and can be costly to get data on emission factors that can be used for carbon accounting. Electricity Maps want to enable companies of all sizes to do more granular scope 2 carbon accounting and to track emissions more accurately.

**What is included in the data**

The following fields are available in each CSV downloaded from the data portal in time aggregations of hourly, daily, monthly or yearly: 

- Datetime (UTC)
- Country
- Zone Name
- Zone Id
- Carbon Intensity gCO₂eq/kWh (direct)
- Carbon Intensity gCO₂eq/kWh (life cycle assessment)
- Low Carbon Percentage
- Renewable Percentage
- Data Source
- Data Estimated
- Data Estimation Method
- The data available is for 2021 and 2022. The data for 2023 will be available in January 2024 - click here to view the data portal and sign up to get notified when the data is live.

**Comment by NAMRATA-WOKE** [comment]
Thanks for sharing @Celineleopoul.
